src/dags/spoorlijnen.py

```diff
@@ -0,0 +1,212 @@
+import operator
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.operators.postgres_operator import PostgresOperator
+
+from provenance_rename_operator import ProvenanceRenameOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+from swift_operator import SwiftOperator
+
+from common import (
+    default_args,
+    pg_params,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    MessageOperator,
+)
+
+from postgres_check_operator import (
+    PostgresMultiCheckOperator,
+    COUNT_CHECK,
+    GEO_CHECK,
+)
+
+dag_id = "spoorlijnen"
+variables_spoorlijnen = Variable.get("spoorlijnen", deserialize_json=True)
+files_to_download = variables_spoorlijnen["files_to_download"]
+tmp_dir = f"/tmp/{dag_id}"
+total_checks = []
+count_checks = []
+geo_checks = []
+check_name = {}
+
+# needed to put quotes on elements in geotypes for SQL_CHECK_GEO
+
+
+def quote(instr):
+    return f"'{instr}'"
+
+
+with DAG(
+    dag_id,
+    description="spoorlijnen metro en tram",
+    default_args=default_args,
+    user_defined_filters=dict(quote=quote),
+    template_searchpath=["/"],
+) as dag:
+
+    # 1. Post info message on slack
+    slack_at_start = MessageOperator(
+        task_id="slack_at_start",
+        http_conn_id="slack",
+        webhook_token=slack_webhook_token,
+        message=f"Starting {dag_id} ({DATAPUNT_ENVIRONMENT})",
+        username="admin",
+    )
+
+    # 2. Create temp directory to store files
+    mkdir = BashOperator(task_id="mkdir", bash_command=f"mkdir -p {tmp_dir}")
+
+    # 3. Download data
+    download_data = [
+        SwiftOperator(
+            task_id=f"download_{file}",
+            swift_conn_id="SWIFT_DEFAULT",
+            container="spoorlijnen",
+            object_id=f"{file}",
```

Comment:
`str(file)` is beter oplossing ;)

Reply:
Ah ok, dat wist ik niet, ga ik aanpassen